docs/source/guide/ml_create.md

```diff
@@ -1,31 +1,40 @@
 ---
-title: Write your own ML backend
+title: About the Label Studio ML backend
 type: guide
 order: 607
 meta_title: Machine Learning SDK
 meta_description: Set up your machine learning model to output and consume predictions in your data science and data labeling projects. 
 ---
 
+The Label Studio ML server uses [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) and [supervisord](http://supervisord.org/) and handles background training jobs with [RQ](https://python-rq.org/).
+
+
+## Write your own ML backend
+
 Set up a machine learning model as a backend to Label Studio so that you can dynamically output and consume predictions as labeling occurs. You can follow this tutorial to wrap custom machine learning model code with the Label Studio ML SDK, or refer to [example ML backend tutorials](ml_tutorials.html) to integrate with popular machine learning frameworks such as PyTorch, GPT2, and others. 
 
-This example tutorial outlines how to wrap a simple text classifier based on the [scikit-learn](https://scikit-learn.org/) framework with the Label Studio ML SDK in Python. Perform the following steps:
+This example tutorial outlines how to wrap a simple text classifier based on the [scikit-learn](https://scikit-learn.org/) framework with the Label Studio ML SDK in Python. 
+
+Perform the following steps:
 1. Review and follow [the prerequisites](#Prerequisites).
 2. [Declare and initialize a class](#Declare-and-initialize-a-class).
 3. [Make predictions with your ML backend](#Make-predictions-with-your-ML-backend).
 4. [Train a model with your ML backend](#Train-a-model-with-your-ML-backend).
-5. [Specify requirements for your ML backend](#Specify-requirements-for-your-ML-backend)
+5. [Specify requirements for your ML backend](#Specify-requirements-for-your-ML-backend).
 6. [Start running your ML backend](#Start-running-your-ML-backend).
 
 If you want to create an ML backend that you can use for dynamic ML-assisted labeling with interactive pre-annotations, see [Support interactive preannotations in your ML backend](#Support-interactive-preannotations-in-your-ML-backend).
 
-## Prerequisites 
+
+### Prerequisites 
```

Comment:
I'm not sure about changing of #-level on this page.
#### - is too invisible and links to them can't be copied.